build/upload-to-s3.sh

```diff
@@ -7,12 +7,12 @@ function install_aws_cli() {
     aws_dir=$(dirname $(readlink -f $0))
     curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
     unzip -q -o awscliv2.zip
-    ./aws/install -i $aws_dir/aws-cli -b /home/travis/bin
+    
+    # ./aws/install -i $aws_dir/aws-cli -b /home/travis/bin
```

Comment:
Why comment and not remove? It might make the block cleaner to remove if you don't need it.

Reply:
I commented this line out since in a separate PR we will be changing this line. So I commented it out so it will be available for reference.